components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -47,37 +48,66 @@
       event_description,
       "",
     );
+    show_hosts = getPropertyValue(internalProps.show_hosts, show_hosts, "show");
   }
 
-  $: editableProperties = [
-    {
-      label: "Event Title",
-      value: event_title,
-    },
-    {
-      label: "Event Description",
-      value: event_description,
-    },
-  ];
+  $: manifestProperties = {
+    event_title,
+    event_description,
+    show_hosts,
+  };
   // #endregion mount and prop initialization
 
   function saveProperties() {
     console.log("Saving the following properties:");
-    console.log(editableProperties);
+    Object.entries(manifestProperties).forEach(([k, v]) => {
+      console.log(k, v);
+    });
   }
 </script>
 
 <style lang="scss">
+  fieldset {
+    margin: 0;
+    padding: 0;
+    border: 0;
+  }
 </style>
 
 {#if manifest && manifest.error}
   <nylas-domain-error {id} />
 {:else}
-  {#each editableProperties as prop}
+  <fieldset>
+    <label>
+      <strong>Event Title</strong>
+      <input type="text" bind:value={manifestProperties.event_title} />
+    </label>
+  </fieldset>
```

Comment:
I don't think a fieldset is needed here as its meant to group multiple controls. Perhaps the label can have the same styling as a fieldset?

Reply:
Fair point! I do want to use a consistent element, however; I will fall back to a div for this case.